prime-router/src/testIntegration/kotlin/serializers/Hl7SerializerIntegrationTests.kt

```diff
@@ -667,4 +658,23 @@ NTE|1|L|This is a final comment|RE"""
         assertThat(msh43).isEqualTo("correctText-YES!")
         assertThat(msh10).isEqualTo("yeah/success")
     }
+
+    @Test
+    fun `test NTE Source`() {
+        val parser = context.pipeParser
+
+        val uploadStream = File("./src/test/unit_test_files/csv-upload-test.csv").inputStream()
```

Comment:
test suites should have all files within the test suite folders.  Moves this file into the src/testIntegrations/resources/xxx folder.